Topics/Tech_Stacks/GraphQL_Apollo.md

```diff
@@ -0,0 +1,76 @@
+# GraphQL With Apollo
+
+## Table of contents
+### [Introduction](#introduction)
+### [GraphQL Explained](#graphql-explained)
+### [Apollo Explained](#apollo-explained)
+### [Tutorial With Basic Setup](#tutorial-with-basic-setup)
+
+## Introduction
+GraphQL simply put is a query language used to build APIs. It is a framework that is very different from the traditional approach of API design. It allows clients to request exactly what they need, nothing more and nothing less, making it an incredibly efficient tool for data retrieval and manipulation. Used across various applications, from small-scale projects to large enterprise-level systems, GraphQL provides a flexible and efficient way to interact with data. It stands out due to its compatibility with a wide array of technologies, including but not limited to JavaScript frameworks like React, Vue.js, and Angular. Unlike SQL, which is a query language specifically for interacting with relational databases, GraphQL is not tied to any specific database or storage engine. Its agnostic nature allows it to work seamlessly with both relational databases like MySQL and PostgreSQL, as well as non-relational databases like MongoDB. At its core, GraphQL operates by defining a schema that describes all possible data points and relationships in the system, enabling clients to query precisely what they need, leading to optimized data loading and reduced network usage.
```

Comment:
Terminology: On line 10 in "GraphQL simply put is a query language used to build APIs. It is a framework that...", I think you shouldn't call GraphQL a framework. Unlike frameworks, which provide the structure and guidelines for building applications, GraphQL focuses solely on the way to query and manipulate data.